### PR TITLE
[MIG][12.0] fleet vehicle service kanban: Backport 13.0 to 12.0

### PR DESCRIPTION
**Pull request by mymage**
Backportig from 13.0

### CODE REVIEW

**Comment by mymage** [comment]
@ivantodorovich I have some problem on fleet_vehicle_log_services_views with \<activity\> view.
I commented the code to pass Travis.
I googled for activity view on v12 and if I understood it is not available like on v13.
Do you confirm that or do you have some reference do suggest?

I disabled \<widget\> section on fleet_vehicle_log_services_view_form to avoid runbot error.

**Comment by mymage** [comment]
@ivantodorovich As before, I commented the content of \<activity\> and for this I don't found help.
I commented \<widget\> too and I think that can be substituted with a label.

File: fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml [hunk 1]
```diff
@@ -17,12 +17,14 @@
...
                 <div class="oe_button_box" name="button_box">
                 </div>
+                <!-- disabled because generate an error on 12.0
                 <widget
                     name="web_ribbon"
                     title="Archived"
                     bg_color="bg-danger"
                     attrs="{'invisible': [('active', '=', True)]}"
                 />
+                -->
```
Comment (ivantodorovich) [inline fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml]: In 12.0 you can add a toggle active button: https://github.com/odoo/odoo/blob/c881e3fe2a8b3db43fa9613a52b996fdfab21392/addons/account/views/account_view.xml#L225

File: fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml [hunk 2]
```diff
@@ -113,6 +116,7 @@
                         </div>
                     </div>
                 </templates>
+                -->
```
Comment (ivantodorovich) [inline fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml]: You can probably do something like it's done for project. Tasks in 12.0: https://github.com/odoo/odoo/blob/c881e3fe2a8b3db43fa9613a52b996fdfab21392/addons/project/views/project_views.xml#L621

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml (hunk 1 above)
Comment (mymage) [reply fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml]: I did the change, hope it is ok.

File: fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml (hunk 2 above)
Comment (mymage) [reply fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml]: I did the change but I'm not sure that it is good because how to test it.

**Comment by mymage** [comment]
@ivantodorovich There is another problem: I don't know why "Activity Types" menu don't show (and why it is double on v 13.0)

**Comment by ivantodorovich** [comment]
Hi @mymage 

the menu is shown only in debug mode, because it has the base.group_no_one group.

image

There are 2 menues because 1 is coming from the standard `fleet` module, and those are activity types for vehicles.

And the other one is the one in this module: activity types for services


I've tested and it works ok! good job!

**Review by ivantodorovich — APPROVED** [review]

**Comment by mymage** [comment]
> the menu is shown only in debug mode, because it has the base.group_no_one group.

Well, learned one more thing :-)

> I've tested and it works ok! good job!

Thank you, now I will proceed with the last one when this will be merged.

**Comment by mymage** [comment]
@ivantodorovich ping ;-) to merge this to complete #11

**Comment by ivantodorovich** [comment]
We need another member to approve the PR before merging :sweat_smile:

**Comment by mymage** [comment]
@mamcode can you please give a look/review to this PR? Thanks

**Review by mamcode — APPROVED** [review]
LGTM

**Comment by mamcode** [comment]
> @mamcode can you please give a look/review to this PR? Thanks

Done, thanks for backport.

**Pull request merged**